mvr/runtime/mangata-polkadot/src/lib.rs

```diff
@@ -148,6 +149,22 @@ impl WeightToFeePolynomial for WeightToFee {
 	}
 }
 
+pub const DOT_MGX_SCALE_FACTOR_UNADJUSTED: u128 = 10_000_000_000_u128; // 10_000 as DOT/MGX, with 6 decimals accounted for (12 - DOT, 18 - MGX)
+
+pub fn base_tx_in_mgx() -> Balance {
+	UNIT
+}
+
+pub fn mgx_per_second() -> u128 {
+	let base_weight = Balance::from(ExtrinsicBaseWeight::get().ref_time());
+	let base_per_second = (WEIGHT_REF_TIME_PER_SECOND / base_weight as u64) as u128;
+	base_per_second * base_tx_in_mgx()
+}
+
+pub fn dot_per_second() -> u128 {
+	mgx_per_second() / DOT_MGX_SCALE_FACTOR_UNADJUSTED as u128
+}
+
```

Comment:
what is the token setup in the MVR ?
similar to what we have now ? 0 native, 4 relay
which pays onchain fees ?

Reply:
yep, i kept the same ids as on rococo/kusama